src/viz-parole-revocation/VizParoleRevocation.js

```diff
@@ -18,6 +26,30 @@ const VizParoleRevocationWrapper = styled.div`
   width: 100%;
 `;
 
+const BreakdownBarWrapper = styled.div`
+  height: ${(props) => props.height}px;
+  padding-bottom: 16px;
+  position: relative;
+  z-index: ${(props) => props.theme.zIndex.base + props.stackOrder};
+`;
+
+function Breakdowns({ data, dimension }) {
+  const breakdownHeight = HEIGHT / data.size;
+  return Array.from(data, ([key, value], i) => (
+    <BreakdownBarWrapper
+      key={key}
+      height={breakdownHeight}
+      stackOrder={data.size - i}
```

Comment:
@sprad this `stackOrder` business does indeed fix that problem with the tooltip

Reply:
🎉